src/components/ProductDetailPage/ProductDetailPage.tsx

```diff
@@ -76,9 +76,13 @@ export const ProductDetailPage = ({ componentId = 'v1' }: IProductDetailPageProp
         </ContentBox>
 
         <Toolbar>
-          <ToolbarItem>
+          <ToolbarItem reservedWidth>
```

Comment:
How this would be implemented if more `ToolbarItem` is used (for example see `ArtifactsList.tsx`)?

Reply:
@matedo1  it also works well, if you want to reserve width to allow suffix components/buttons not to be wrapped, just add `reserveWidth` to the current component. see pictures:
![image](https://github.com/project-ncl/pnc-web-ui-react/assets/6870455/9be5ceec-5380-467f-90fa-2810a9a1de4b)
![image](https://github.com/project-ncl/pnc-web-ui-react/assets/6870455/cbc0247a-1274-4142-8897-0ba06e8e3129)

Reply:
![image](https://github.com/project-ncl/pnc-web-ui-react/assets/6870455/6fd7fd69-11d4-433f-868a-92b136551165)